src/Parser/Core/Modules/Others/DistanceMoved.js

```diff
@@ -6,12 +6,14 @@ import { formatThousands } from 'common/format';
 import Analyzer from 'Parser/Core/Analyzer';
 
 import SmallStatisticBox, { STATISTIC_ORDER } from 'Main/SmallStatisticBox';
+import { formatPercentage } from "../../../../common/format";
 
```

Comment:
Should be imported like so ``common/format``

Reply:
Thanks for noticing, fixed.